pkg/policy/l3.go

```diff
@@ -32,9 +33,11 @@ import (
 // L3PolicyMap does no locking internally, so the user is responsible for synchronizing
 // between multiple threads when applicable.
 type L3PolicyMap struct {
-	Map       map[string]net.IPNet // Allowed L3 prefixes
-	IPv6Count int                  // Count of IPv6 prefixes in 'Map'
-	IPv4Count int                  // Count of IPv4 prefixes in 'Map'
+	Map              map[string]net.IPNet // Allowed L3 prefixes
+	IPv6Count        int                  // Count of IPv6 prefixes in 'Map'
+	IPv4Count        int                  // Count of IPv4 prefixes in 'Map'
+	SourceRuleLabels labels.Labels        // The User rule labels that resulted in this Map. Can be empty
+	SourceRuleCount  int64                // Number of rules that resulted in this Map
```

Comment:
Out of curiosity, why is the count important?

Reply:
I added this once I realised that many rules will have no labels, so it was impossible to get a quick count of rules affecting an endpoint. This might be a bad choice, though, if we prefer to have the user only ever use `cilium policy` to get policies or their counts. From some quick testing with `cilium policy get` if you pass no labels you get all rules, not ones without labels :/